README.md

```diff
@@ -0,0 +1,52 @@
+
+                HacktoberFest Starter Project
```

Comment:
Please read Github Markdown guide (https://guides.github.com/features/mastering-markdown/). This specifically requires `#` as prefix (also remove all the spaces).